model_hub/model_hub/mmdetection/_callbacks.py

```diff
@@ -140,7 +140,7 @@ def __init__(self, context: det_torch.PyTorchTrialContext, model: torch.nn.Modul
         self.model = model
 
     def on_validation_start(self) -> None:
-        if self.context.distributed.get_size() > 1:
+        if self.context.distributed.get_size() > 1 and self.context._distributed_backend.use_horovod():
```

Comment:
i think we should have logic to switch between what broadcast operation to use; either torch.distributed.broadcast or hvd's broadcast.